libcalico-go/lib/backend/model/block_affinity.go

```diff
@@ -15,13 +15,19 @@
 package model
 
 import (
+	"crypto/sha3"
+	"encoding/base32"
 	"fmt"
 	"reflect"
 	"regexp"
+	"strconv"
 	"strings"
 
+	v4 "github.com/projectcalico/api/pkg/apis/projectcalico/v3"
```

Comment:
v4? 🤔

Reply:
I've been using `apiv3` (or just `v3`) and `libapiv3` to distinguish in my CRD rework PR, as part of making `projectcalico/api/pkg/apis/projectcalico/v3` the canonical API definition.

Reply:
:laughing: Not conscious of writing this